sailfishusermanager.cpp

```diff
@@ -45,6 +45,7 @@ const auto ENVIRONMENT_FILE = QStringLiteral("/etc/environment");
 const QByteArray LAST_LOGIN_UID_KEY("LAST_LOGIN_UID=");
 const int MAX_USERNAME_LENGTH = 20;
 const auto USER_ENVIRONMENT_DIR = QStringLiteral("/home/.system/var/lib/environment/%1");
+const auto USER_REMOVE_SCRIPT_DIR = QStringLiteral("/usr/share/user-manager/removal");
```

Comment:
How about `/usr/share/user-managerd/remove.d`? I'm just thinking that we use names like `user-managerd` (the repo) and `usermanager` (e.g. D-Bus service) but not `user-manager` so I would use either of those. Name of the repo feels more natural to me here but either works. And that `removal` felt a bit out of place so since the interface is `removeUser()` maybe we should maybe mirror that so either `remove` or `removeUser`. I added `.d` to the end because it feels like convention to name those directories that way.